.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/whisperImporter/whisperReader/main
+/cmd/whisperImporter/chunkWriter/main
```

Comment:
I like the idea of putting the `archive` together with it's encoding/decoding methods into `mdata/chunk/archive` and importing it from there. I'll do that